<div class="container py-5">
  <div class="row">
    <div class="col-12 col-lg-10 mx-auto">
      <h2>グループ作成</h2>
      <% if @group.errors.any? %>
        <%= @group.errors.count %>件のエラーが発生しました
        <ul>
          <% @group.errors.full_messages.each do |message| %>
            <li><%= message %></li>
          <% end %>
        </ul>
      <% end %>
      <%= form_with model: @group, url: groups_path do |f| %>
        <div class="form-group">
          <%= f.label :name, 'グループ名' %>
          <%= f.text_field :name, class: 'form-control' %>
        </div>
        <div class="form-group">
          <%= f.label :introduction, 'グループ説明' %>
          <%= f.text_area :introduction, rows: 5, class: 'form-control' %>
        </div>
        <div class="form-group">
          <div class="row">
            <div class="col-md-6 mx-auto">
              <%= f.submit 'グループ作成', class: 'btn btn-outline-secondary btn-block' %>
            </div>
          </div>
        </div>
      <% end %>
    </div>
  </div>
</div>
